### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by fezzopro**
## PR Details

This pull request contains:

- User View connected to User Controller
- Post View connected to Post Controller
- Uses layouts and templates for shared content

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,28 @@
...
+      <span class='flex flex-column user-blog-posts'>Number of Posts: <%= @user.post_counter || 0 %></span>
+    </div>
+  </div>
+  <div class="flex flex-column">
+  Bio:
+    <p>
+      <%= @user.bio%>
+    </p>
+  </div>
+  <% @posts.each do |post| %>
```
Comment (tugsem) [inline app/views/users/show.html.erb:16]: - [ ] Please use the recent posts method that you have created for the user model as required.

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,45 @@
...
+          <%= post.text %>
+          <div class="flex flex-row user-blog-posts">
+            <span>Comments: <%= post.comments_counter || 0 %></span>
+            <span>Likes: <%= post.likes_counter || 0 %></span>
+          </div>
+        </div>
+      </li>
+      <ul class="flex flex-column users-list">
+        <% if post.recent_comments.any? %>
+          <% post.comments.each do |comment| %>
```
Comment (tugsem) [inline app/views/posts/index.html.erb:31]: - [ ] Please use the recent comments method that you created for the post model here as requested.

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @fezzopro  👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️views created as the given wireframe
✔️actions created for the user and post controller
✔️professional documentation
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 1 above)
Comment (fezzopro) [reply app/views/users/show.html.erb:16]: Than you @tugsem for spotting this

Comment (fezzopro) [reply app/views/users/show.html.erb:16]: But as you can see in the UsersController our show message is getting only the recent posts

image

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (fezzopro) [reply app/views/posts/index.html.erb:31]: Thank you for spotting this bug. 
The default limit for recent_comments is set to 5, however, you can change that when calling the recent_comment action `post.recent_comments(limit)`

**Review by tugsem — APPROVED** [review]
## STATUS: APPROVED 👍🎯  
Hello @fezzopro 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⭐ 
✔️ required methods added
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**